sdk/translation/Azure.AI.Translation.Document/src/DocumentTranslateContent.Serialization.cs

```diff
@@ -57,7 +57,7 @@ internal virtual MultipartFormDataRequestContent ToMultipartRequestContent()
                         continue;
                     }
 #if NET6_0_OR_GREATER
-				writer.WriteRawValue(global::System.BinaryData.FromStream(item));
+				writer.WriteRawValue(global::System.BinaryData.FromStream(item.Content));
```

Comment:
This one, we'll want to get Michael to double-check for us.

Reply:
CC: @m-nash